src/sentry/grouping/enhancer/enhancement-configs/newstyle@2023-01-11.txt

```diff
@@ -151,7 +151,7 @@ family:native package:libAudioToolboxUtility.dylib category=av
 family:native function:std::_* category=internals
 family:native function:__swift_* category=internals
 family:native function:block_destroy_helper* category=internals
-family:native function:*_block_invoke* category=internals
+family:native function:*_block_invoke* app:no category=internals
```

Comment:
I will be porting the change to #61603 since this PR is built on a different branch and I want to trim down the events in the test.